.github/workflows/android_test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -ex
+
+rustup target install x86_64-linux-android
+cargo install cargo-apk
+cd "$GITHUB_WORKSPACE/ndk-examples"
+cargo apk run --example hello_world --target x86_64-linux-android
+
+sleep 30s
+
+adb logcat *:E RustStdoutStderr:V -d > ~/logcat.log
+
+if grep 'RustStdoutStderr' ~/logcat.log;
+then
+    echo "App running"
+else
+    exit 1
+fi
+
+if grep -e 'thread.*panicked at' ~/logcat.log;
+then
+    exit 1
+else
+    exit 0
+fi
```

Comment:
Should these `echo ::error::something` so that the failure reason can be read from a CI "annotation" instead of having to download+grep the logs?

Reply:
That could work as well, but having at least the `panicked at` line with the panic reason in an `::error::` creates a nice annotation in CI run "overview".

https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-error-message

Reply:
Perhaps it's worth experimenting some with these, introducing an intentional panic and seeing what is printed in the final output.

You can possibly substitute this entire if block with:

```sh
! (grep -e 'thread.*panicked at' ~/logcat.log | sed 's/^/::error::/')
```

Reply:
Is it good enough as is to merge?

Reply:
@dvc94ch Sorry, completely forgot about this PR. I will fix this and will squash commits.

Reply:
@Gordon-F Yeah I guess that works well enough. Bonus would be to include the stacktrace, but that's more annoying to parse and it's probably more relevant to download and read the whole logs from artifacts in that case.

Note that the test currently isn't working, can you look into that? It looks like the app hasn't started at all.

Reply:
Used a wrong tag. Fixed now.